src/shared/utils.ts

```diff
@@ -0,0 +1,29 @@
+import Amplify from "aws-amplify";
+import { AuthState, AmplifyConfig } from "shared/interfaces/amplify.interface";
+
+export const isAuthenticated = (state: string) => {
+  return state === AuthState.SIGNED_IN;
+};
+
+/*
+  Config is used to initialize application with Amplify
+*/
+export class Config extends Amplify {
+  private static instance: Config;
+
+  private constructor() {
+    super();
+  }
+
+  public static getInstance(): Config {
+    if (!Config.instance) {
+      Config.instance = new Config();
+    }
+    return Config.instance;
+  }
+
+  public init(amplifyConfig: AmplifyConfig) {
+    Config.configure(amplifyConfig);
+    Config.I18n.setLanguage(amplifyConfig.language || 'us');
+  }
+}
```

Comment:
This Singleton is used to abstract configuration logic from the root component by making it smaller and clearer.
Instead of calling in root component:
```ts
import Amplify, {I18n, ....} from "aws-amplify";
Amplify.configure(config);
I18n.setLanguage('fr');
...
```

We just call 
```ts
import { Config } from "shared/utils";
Config().getInstance().init(config);
```